src/jit/emit.h

```diff
@@ -932,7 +932,7 @@ class   emitter
             }
             void            iiaSetInstrCount(int count)
             {
-                assert(abs(count < 10));
+                assert(count < 10);
```

Comment:
Yeah clang was throwing a warning here too, I can however reflect @CarolEidt s suggestion if that is correct, there were a couple of cases like this.